### PR TITLE
fix: PR body comment for pinning is wrong/outdated

### LINKED ISSUE #10837: PR body comment for pinning is wrong/outdated
**How are you running Renovate?**

- [x] WhiteSource Renovate hosted app on github.com
- [ ] Self hosted

If using the hosted app, please skip to the next section.
Otherwise, if self-hosted, please complete the following:

Please select which platform you are using:

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server
- [ ] Gitea
- [x] github.com
- [ ] GitHub Enterprise Server
- [ ] gitlab.com
- [ ] GitLab self-hosted

Renovate version: ...

**Describe the bug**

@rarkins noted while reviewing PR #10835 that:

> Also I noted that the pr body comment for pinning is wrong/outdated but it's unrelated to this PR

I'm making a new issue so we track this problem. 😄 

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

@rarkins I'm willing to make the reproduction repo, with a little help on how to trigger the bad behavior. 😉

### PR DESCRIPTION
**Pull request by MaronHatoum**
## Changes
removed pin prBodyExtra + fixed unit test

## Context
Closes https://github.com/renovatebot/renovate/issues/10837

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/update/pr/body/notes.ts [hunk 1]
```diff
@@ -38,11 +38,5 @@ export function getPrExtraNotes(config: BranchConfig): string {
     );
   }
 
-  if (config.isPin) {
-    res += emojify(
-      `:pushpin: **Important**: Renovate will wait until you have merged this Pin PR before creating any *upgrade* PRs for the affected packages. Add the preset \`:preserveSemverRanges\` to your config if you instead don't wish to pin dependencies.\n\n`
```
Comment (HonkingGoose) [inline lib/workers/repository/update/pr/body/notes.ts:43]: Do we want to keep the note about using `:preserveSemverRanges`?

Comment (rarkins) [reply lib/workers/repository/update/pr/body/notes.ts:43]: Good idea, yes

Comment (MaronHatoum) [reply lib/workers/repository/update/pr/body/notes.ts:43]: @HonkingGoose  @rarkins Like this ?
image

Comment (MaronHatoum) [reply lib/workers/repository/update/pr/body/notes.ts:43]: @HonkingGoose @rarkins like this?
image

File: lib/workers/repository/update/pr/body/notes.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export function getPrExtraNotes(config: BranchConfig): string {
 
   if (config.isPin) {
     res += emojify(
-      `:pushpin: **Important**: Renovate will wait until you have merged this Pin PR before creating any *upgrade* PRs for the affected packages. Add the preset \`:preserveSemverRanges\` to your config if you instead don't wish to pin dependencies.\n\n`
+      `Add the preset \`:preserveSemverRanges\` to your config if you instead don't wish to pin dependencies.\n\n`
```
Comment (HonkingGoose) [inline lib/workers/repository/update/pr/body/notes.ts:43]:
```suggestion
      `Add the preset \`:preserveSemverRanges\` to your config if you don't want to pin your dependencies.\n\n`
```

Let's improve the message as well. 😉

File: lib/workers/repository/update/pr/__snapshots__/index.spec.ts.snap [hunk 3]
```diff
@@ -64,7 +64,7 @@ Array [
...
... Package | Type | Update | Change |\\n|---|---|---|---|\\n| [dummy](https://dummy.com) ([source](https://github.com/renovateapp/dummy), [changelog](https://github.com/renovateapp/dummy/changelog.md)) | devDependencies | pin | \`1.0.0\` -> \`1.1.0\` |\\n\\nAdd the preset \`:preserveSemverRanges\` to your config if you instead don't wish to pin dependencies.\\n\\n---\\n\\n### Release Notes\\n\\n<details>\\n<summary>renovateapp/dummy</summary>\\n\\n### [\`v1.1.0\`](https://github.com/renovateapp/dummy/compare/v1.0.0...v1.1.0)\\n\\n[Compare Source](https://github.com/renovateapp/dummy/compare/v1.0.0...v1.1.0)\\n\\n</details>\\n\\n---\\n\\n### Configuration\\n\\n📅 **Schedule**: \\"before 5am\\" in timezone some timezone.\\n\\n🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.\\n\\n♻ **Rebasing**: Whenever PR is behind base branch, or you tick the rebase/retry checkbox.\\n\\n🔕 **Ignore**: Close this PR and you won't be reminded about this update again.\\n\\n---\\n\\n - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, click this checkbox.\\n\\n---\\n\\nThis PR has been generated by [Renovate Bot](https://github.com/renovatebot/renovate).",
```
Comment (HonkingGoose) [inline lib/workers/repository/update/pr/__snapshots__/index.spec.ts.snap:67]: I guess the Jest snapshot needs to be updated as well?

Comment (MaronHatoum) [reply lib/workers/repository/update/pr/__snapshots__/index.spec.ts.snap:67]: you are right

File: lib/workers/repository/update/pr/index.spec.ts [hunk 4]
```diff
@@ -473,9 +473,6 @@ describe('workers/repository/update/pr/index', () => {
           sourceBranch: 'renovate/dummy-1.x',
         },
       ]);
-      expect(platform.createPr.mock.calls[0][0].prBody).toContain(
-        'this Pin PR'
```
Comment (rarkins) [inline lib/workers/repository/update/pr/index.spec.ts:477]: Revert this removal and instead change the text to match something in the new string

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by HonkingGoose — APPROVED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**